ens/utils.py

```diff
@@ -108,16 +109,30 @@ def is_valid_name(name):
         return False
 
 
-def label_to_name(label, default_tld, recognized_tlds):
+def label_to_name(label, recognized_tlds):
     label = normalize_name(label)
     pieces = label.split('.')
     if pieces[-1] not in recognized_tlds:
-        pieces.append(default_tld)
+        raise InvalidTLD(
+            f"The label: {label} has an unsupported TLD of {pieces[-1]}. "
+            f"ENS.py by default supports the following TLDs: {recognized_tlds}. "
+            "If you'd like to use an unsupported TLD, please set the environment variable: "
+            "'ENS_RECOGNIZED_TLDS' to a string of desired TLDs separated by a colon (:)."
+        )
     return '.'.join(pieces)
 
 
 def dot_eth_name(label):
-    return label_to_name(label, 'eth', RECOGNIZED_TLDS)
+    recognized_tlds = get_recognized_tlds()
```

Comment:
I don't think `dot_eth_name` should exist anymore. Looks like it can be replaced by `normalize_name` everywhere. Then this and `label_to_name` can be dropped entirely.